redis.conf

```diff
@@ -110,6 +110,28 @@ bind 127.0.0.1 -::1
 # even if no authentication is configured.
 protected-mode yes
 
+# In order to make Redis more secure by default and reduce the attack surface
+# on innocent users. By default, some sensitive configs are immutable, and some
+# sensitive commands are blocked.
+#
+# The protected configs are ones that users don't normally need to modify at
+# runtime and control which files Redis writes to (e.g. 'dir', 'dbfilename')
+#
+# The protected commands are ones that users don't normally need to use, and
+# are considered to increase the attack surface of Redis.
+#
+# These can be exposed to either all connections or just local ones by setting
+# each of the configs listed below to either of these values:
+#
+# no    - Block for any connection (remain immutable)
+# yes   - Allow for any connection (no protection)
+# local - Allow only for local local connections. Ones originating from the
+#         IPv4 address (127.0.0.1), IPv6 address (::1) or Unix domain sockets.
+#
+# enable-protected-configs no
+# enable-debug-command no
+# enable-module-command no
```

Comment:
This isn't consistent with the config values in config.c, `enable-dubug-command` and `enable-module-comand` are defaulted to local.  I'm not sure I understand why the default for `enable-debug-command` is local, but I am okay with it if that was the consensus. If the correct values are in config.c, it should be reflected here and more clearly documented.

Reply:
i missed it.
I think they should all be default to `no` (being more secure by default).
note that for some users who use proxy or other forms of redirection, external traffic is coming from a local connection.

Reply:
Oh, the proxy point is good one I didn't consider. I agree then we should keep the default values as no then.